python/astro_metadata_translator/properties.py

```diff
@@ -46,6 +46,8 @@
               "detector_num": ("Unique (for instrument) integer identifier for the sensor.", "int"),
               "detector_name": ("Name of the detector within the instrument (might not be unique).",
                                 "str"),
+              "detector_unique_name": ("Unique name of the detector, generally combining group with name",
```

Comment:
I think the term slot is better than detector here.  I.e. the detector could be swapped out, but that location in the camera will still exist.  Serial is to break that degeneracy.

Reply:
Would it help if I changed the word to "unique name of the detector within the focal plane,"?
ie not implying that detector_serial is superseded.

Reply:
Sure.  I don't want to make a bigger deal over it than it is.